DQM/SiPixelPhase1Common/src/SiPixelCoordinates.cc

```diff
@@ -0,0 +1,916 @@
+// -*- C++ -*-
```

Comment:
@schneiml 

thanks
Sounds like it belongs in a data format (tracker already has much of this sort of functionality in a data format) or in a reco package.

what is the impact on memory of this cache?